.husky/pre-commit

```diff
@@ -1,3 +1,3 @@
 yarn --cwd packages/airnode-validator/ run docs
-yarn prettier --write "./**/*.md" --loglevel silent
+yarn prettier --write "./packages/airnode-validator/**/*.md" --loglevel silent
```

Comment:
Why do we write anything during the pre-commit hook? Pre-commit hook should make you aware that there is a problem and not allow you to commit unless you solve it. It shouldn't silently change the content of files. That should always be a conscious decision of a developer.

Reply:
Because validator docs are genereated (and written on disk) on pre commit hook, but thoes files are not correctly formatted.

But we could make it a pre-push hook, wdyt?

Reply:
I kinda like making the prettier check into something more conscious for the developer instead of doing it automatically. I think we need to remove the `yarn run pretty-quick` below though?

I also generally prefer pre-commit to pre-push but that's just my personal preference

Reply:
I am confused. What do you suggest?

Reply:
Sorry I think I may have misunderstood myself. This should probably be kept as is if the files are generated during pre-commit. Ignore me 😬